gef.py

```diff
@@ -3079,7 +3092,7 @@ def mprotect_asm(cls, addr: int, size: int, perm) -> str:
 
 @register_architecture
 class MIPS64(MIPS):
-    aliases = ("MIPS64")
+    aliases = ("MIPS64",)
```

Comment:
hint missing here or redundant on other arches

Reply:
i think here it wasn't needed because there was no discrepancy with the super class